src/dictionary/api.ts

```diff
@@ -80,16 +80,26 @@ export const get = async (req: Request, res: Response) => {
   return res.status(200).send(schema);
 };
 
+export const getClinicalSchemas = async (withFields: boolean) =>
```

Comment:
This function has always returned either [String] or [{ }]; this does not play nicely w/ GraphQL TypeDefs
Open to suggestions + solutions

Reply:
Refactored for reusability (API vs GQL)
API is currently converting Clinical response to JSON, which masks difference btwn `[String] or [{ }]`